data/mods/My_Sweet_Cataclysm/sweet_monsters.json

```
@@ -251,9 +251,24 @@
     "anger_triggers": [ "HURT" ],
     "death_function": [ "NORMAL" ],
     "harvest": "gummy_bear",
-    "reproduction": { "baby_egg": "candy3", "baby_count": 1, "baby_timer": 10 },
+    "reproduction": { "baby_egg": "candy3gator", "baby_count": 1, "baby_timer": 10 },
```

```suggestion
    "reproduction": { "baby_egg": "candy3", "baby_count": 1, "baby_timer": 10 },
```
Wrong search and replace I guess, gummy bear should not make gators X)